[2022-04-11 13:44:58,751] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: download_data_dag.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-04-11 13:44:58,762] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: download_data_dag.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-04-11 13:44:58,762] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 13:44:58,762] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-11 13:44:58,762] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 13:44:58,775] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-01 00:00:00+00:00
[2022-04-11 13:44:58,785] {standard_task_runner.py:52} INFO - Started process 325168 to run task
[2022-04-11 13:44:58,791] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'download_data_dag', 'local_to_gcs_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '1668', '--raw', '--subdir', 'DAGS_FOLDER/nfl_to_gcs.py', '--cfg-path', '/tmp/tmpa9azxby9', '--error-file', '/tmp/tmptqecukh_']
[2022-04-11 13:44:58,792] {standard_task_runner.py:77} INFO - Job 1668: Subtask local_to_gcs_task
[2022-04-11 13:44:58,864] {logging_mixin.py:109} INFO - Running <TaskInstance: download_data_dag.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 4ed3eda454fd
[2022-04-11 13:44:58,939] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=download_data_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-04-11 13:44:59,432] {python.py:175} INFO - Done. Returned value was: None
[2022-04-11 13:44:59,457] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=download_data_dag, task_id=local_to_gcs_task, execution_date=20190101T000000, start_date=20220411T134458, end_date=20220411T134459
[2022-04-11 13:44:59,529] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-11 13:44:59,593] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
